src/lazy/value.rs

```diff
@@ -496,21 +496,21 @@ mod tests {
     #[case::typed_null("null.list", IonType::List)]
     #[case::annotated_typed_null("foo::null.list", IonType::List.with_annotations(["foo"]))]
     #[case::boolean("false", false)]
-    #[case::negative_int("-1", -1)]
+    #[case::negative_int("-1", - 1)]
```

Comment:
At a glance, the whitespace changes in this file don't seem like something `rustfmt` would require. Did your IDE add these? I'm ok with it if it's a `rustfmt` thing.

Reply:
Ahh. I think yes, it was added by IDE. I will check with `rustfmt` and add change accordingly.